paper_trail.gemspec

```diff
@@ -28,7 +28,7 @@ has been destroyed.
   s.required_ruby_version = ">= 2.3.0"
 
   # Rails does not follow semver, makes breaking changes in minor versions.
-  s.add_dependency "activerecord", [">= 4.2", "< 5.3"]
+  s.add_dependency "activerecord", [">= 4.2", "< 6.1"]
```

Comment:
I'd be more comfortable with `< 6.0` here, as there will probably be more breaking changes before 6.0 is final.

Reply:
with constrain 6.0 we can't install even alpha rails version

Reply:
@jaredbeck is right, `< 6.0` allow us to support alpha, beta and release candidates without supporting final release.

```ruby
Gem::Version.new('6.0') > Gem::Version.new('6.0.0.alpha')
# true
```